packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -899,6 +900,40 @@ const Conversation = WebexPlugin.extend({
 
     // triggers user-activity to reset logout timer
     this.webex.trigger('user-activity');
+    if (activity.target && !activity.target.defaultActivityEncryptionKeyUrl) {
```

Comment:
With the changes implemented here, we will need to amend an integration test to validate that the changes implemented work as expected.

Reply:
Okay, will update integration test for it. But @InteractiveTimmy I am unable to run test locally receiving timeout error. 
![image](https://user-images.githubusercontent.com/11908216/73237061-74f29980-41ba-11ea-8b6b-386e79799ffc.png)
I used following command :
```
LOGGER_LEVEL=log DEBUG=messages npm test -- --packages @webex/internal-plugin-conversation --node --integration
```
Could you confirm the environment in which webex-js-sdk run on circle ci? Also, whether above command seem correct for running test locally.

Reply:
Your test results should be viewable from the [checks](https://github.com/webex/webex-js-sdk/pull/1477/checks) page. I'm confirming with the team is there's an alternative configuration that can be used to streamline local tests. I will update this as soon as I can validate this.

Reply:
Thanks @InteractiveTimmy , it will really help to add new tests and update existing tests if we can streamline local tests.